src/test/java/org/fundacionjala/coding/omar/Kata10Test.java

```diff
@@ -0,0 +1,111 @@
+package org.fundacionjala.coding.omar;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Powered by Omar Limbert Huanca Sanchez - AT06
+ * This class contains UnitTests for Kata10.java.
+ */
+public class Kata10Test {
+
+    private Kata10 highestAndLowest;
+
+    @Before
+    public void Kata10Test() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this method to prevent any misunderstanding or make it a constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1223)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)